coalib/bearlib/languages/definitions/shell.py

```diff
@@ -0,0 +1,18 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class shell:
+    aliases = 'bash', 'sh'
+
+    extensions = '.sh', '.bash', '.zsh'
+    comment_delimiter = '#'
+    multiline_comment_delimiters = {': '': '''}
```

Comment:
I dont understand this one?

Reply:
```
>>> a = {': '': '''}
>>> type(a)
<class 'set'>
>>> len(a)
1
```

Reply:
@jayvdb shell allows for multi line comments like this: https://stackoverflow.com/a/17574894

Reply:
>>> a.pop()
': : '
```

Reply:
see other definitions of `multiline_comment_delimiters`

Reply:
also would be good to include [this multiline approach](https://stackoverflow.com/a/19182350/5037965)